app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+    def all(self) -> list[Actor]:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [
+            Actor(*row) for row in actors_cursor
+        ]
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            "(first_name, last_name) VALUES (?, ?) ",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def delete(self, id: int) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self.table_name} WHERE actor_id=? ",

```

Comment:
Using f-strings to construct SQL queries can lead to SQL injection vulnerabilities. Consider using parameterized queries to prevent this issue.